scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -0,0 +1,46 @@
+from unittest import mock
+import os
+import logging
+import yaml
+
+from lib.cmds import cmd_ansible
+
+log = logging.getLogger(__name__)
+
+
+@mock.patch("lib.process_manager.subprocess_run")
+def test_cmd_ansible(subprocess_run, tmpdir):
+    '''
+    This test coverage overlap with tests from
```

Comment:
ok they overlap, so why to add them? Please add the purpose of the test in this docstring